lib/lint/formatters/asFormatter.js

```diff
@@ -41,10 +41,10 @@ class Formatter extends abstractFormatter_1.AbstractFormatter {
         });
     }
 }
+exports.Formatter = Formatter;
 Formatter.metadata = {
     formatterName: "as",
     description: "AssemblyScript's TSLint formatter.",
     sample: "Similar to ASC's output.",
     consumer: "human",
 };
-exports.Formatter = Formatter;
```

Comment:
nit: any reason to move this upper?

Reply:
I didn't move it, I just ran `npm run build` in `lib/lint`.

Reply:
Got it

Reply:
I guess it doesn't make a difference. It probably depends on the typescript version used for compilation. I have tsc 3.6.3 under node_modules.